src/app/services/map-view.service.ts

```diff
@@ -20,10 +22,24 @@ export class MapViewService {
 
   // Define base colors
   COLORS = {
-    red: [255, 0, 0, 0.9],
```

Comment:
Why has the alpha been changed from 0.9 to 1?

Reply:
It makes almost no difference on the map, and I guess at some point when experimenting it got changed from 0.9 to 1. We can still tweak those values later if necessary though.